node/service.go

```diff
@@ -74,6 +75,12 @@ func (ctx *ServiceContext) ExtRPCEnabled() bool {
 	return ctx.config.ExtRPCEnabled()
 }
 
+// RPCGasCap returns the global cap for gas allowance on eth-call variants
+// over rpc. This can be used to protect against DoS attacks
+func (ctx *ServiceContext) RPCGasCap() *big.Int {
+	return ctx.config.RPCGasCap
+}
```

Comment:
Please move this out of `node`. The node is a dump thing that start arbitrary services with RPC APIs and networking layer. It should be oblivious to internals of what is actually starts.

Reply:
PS If you move the field into eth.Config, you won't even need this method at all.